### DISCUSSION TITLE
Default home page after login success

### DISCUSSION THREAD

**Discussion by Husnain-QH** [kimai/kimai#3769] [General]
Is there a way to setup the default URL after login is successful?

for e.g. 
setting a URL after SAML login is successful
and
setting a URL after normal login (through login form) is successful

would be very thankful as we want to redirect to a custom page which we have created.

**Comment by kevinpapst** [comment]
Not freely, The user can decide where he is redirected to after login.
But the choices are limited to a couple of internal routes. 
Check the user preferences.
You could probably build something like that with a plugin.
